packages/studioCMS/src/integrations/studioCMSDashboard/index.ts

```diff
@@ -292,6 +301,17 @@ export default defineIntegration({
 					// Log that the setup is complete
 					studioLogger(LoggerOpts.logInfo, DashboardStrings.SetupComplete);
 				},
+				'astro:config:done': ({ injectTypes }) => {
+					// Add Virtual DTS Files
+					injectTypes({
+						filename: 'dashboard.d.ts',
```

Comment:
And prefixing this one

Reply:
https://github.com/astrolicious/studiocms/pull/271#discussion_r1735383527